process/accountProcessor.go

```diff
@@ -114,13 +138,15 @@ func (ap *AccountProcessor) getAccountsInShard(addresses []string, shardID uint3
 	apiPath := addressPath + "bulk"
 	apiPath = common.BuildUrlWithAccountQueryOptions(apiPath, options)
 	for _, observer := range observers {
+		fmt.Printf("getting accounts for shard %d for observer with address %s \n", shardID, observer.Address)
```

Comment:
remove fmt.Printf / fmt.Println here and on L149?

Reply:
done